lib/Sass/common/_variables.scss

```diff
@@ -149,6 +149,8 @@ $chart-text-color: #fff;
 $chart-line-color: #ccc;
 $chart-preview-line-color: $chart-line-color;
 $chart-darker: darken($dark, 8%);
+$chart-panel-background: $dark;
+$chart-panel-header: $dark;
```

Comment:
is this new design from @philipgrimmett ? otherwise the header should have a slight different shade of color from chart body
<img width="686" alt="Screen Shot 2019-04-24 at 10 39 27 am" src="https://user-images.githubusercontent.com/7508939/56624729-71574280-667d-11e9-91d7-6f1548ad0497.png">

Reply:
Fixed :)